microsoft/testsuites/acctests/sgxtest.py

```
@@ -0,0 +1,217 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import os
+import pytest
+import subprocess
+
+from typing import Any
+from pathlib import Path
+from typing import Any
+from assertpy import assert_that
+
+from lisa import (
+    CustomScript,
+    CustomScriptBuilder,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.operating_system import Posix
+from lisa.tools import Echo, Uname
+
+########### HELPER METHODS ###################
+
+@staticmethod
+def getOEInstalledPath():
+    return os.path.expanduser("~/openenclave-install/")
+
+@staticmethod
+def getSGXVersion():
+    if isOeInstalled():
+        openenclave_install_path = getOEInstalledPath()
+        print("!!!!!!!!!!!!!!!!!!!!!!!!!!STARTED")
+        print(openenclave_install_path)
+        stdout = node.execute([openenclave_install_path + "bin/oesgx"], shell=True)
+        oesgx = stdout
+        sgx_version = ""
+        if "CPU supports Software Guard Extensions:SGX1" in oesgx:
+            sgx_version = "sgx1"
+        elif "CPU supports Software Guard Extensions:SGX2" in oesgx:
+            sgx_version = "sgx2"
```

What happened if both of them are not supported? Does it need to raise an exception?